src/constants/app-constants.ts

```diff
@@ -292,3 +294,10 @@ export const TRACKER_TICKET_STATUS_MAP = {
 export const COMMA_SEPARATED_STRING_PATTERN = '([(a-zA-Z])([a-zA-Z-_\'\\s]+,\\s?)*([(a-zA-Z-_\'\\s]+)([(a-zA-Z])';
 
 export const OAUTH_CALLBACK_EVENT_KEY = 'OAuthCallback';
+
+// <-----------------constants for PBKDF2 encryption------------------>
+export const ITERATION_COUNT = 10000;
+
+export const KEY_SIZE = 256 / 32;
+
+export const SALT_FOR_PASSWORD = '';
```

Comment:
Why are we not using salt here?

Reply:
discussed.

Reply:
Please pen down the discussion on the ticket, for future references.

Reply:
done.